catalog/utils.go

```diff
@@ -0,0 +1,23 @@
+package catalog
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/databricks-sdk-go"
+)
+
+func validateMetastoreId(ctx context.Context, w *databricks.WorkspaceClient, metastoreId string) error {
```

Comment:
nit: add a small doccomment here explaining the purpose of this and where it should be added (e.g. all create/update/delete methods for UC resources that accept a metastore_id parameter).